### ISSUE TITLE
computation failed 

### ISSUE THREAD

**Issue by aanaishe** [kassambara/ggpubr#138]
I just updated R Studio (to v 1.1.463) and ggpubr (to v 0.1.9) and code that previously worked now doesn't. When trying to add p-values to my plots using "stat_compare_means()", I now get an error that says "Computation failed in 'stat_compare_means()': argument is of length zero". I tried an simpler line of code using a dataset available in R and a tutorial on ggpubr, and got the same result.

Thanks for the help!

> data("ToothGrowth")
> p <- ggboxplot(ToothGrowth, x = "supp", y = "len",  color = "supp", palette = "npg", add = "jitter")
> p
> p + stat_compare_means()
Warning message:
Computation failed in `stat_compare_means()`:
argument is of length zero 

![image](https://user-images.githubusercontent.com/13001509/48511733-31ae4400-e81d-11e8-8a97-f047bcebb5e5.png)

**Comment by kassambara** [comment]
This has been fixed in the latest developmental version (see discussion: https://github.com/kassambara/ggpubr/issues/135)

Kindly install the latest developmental version from GitHub as follow:

```r
if(!require(devtools)) install.packages("devtools")
devtools::install_github("kassambara/ggpubr")
```

**Comment by aanaishe** [comment]
Ah, I accidentally missed that. Thanks! It works perfectly now.

**Comment by jsan4christ** [comment]
Same problem, unfortunately the fix does not help with R 3.5, any other hacks? 	
Computation failed in `stat_compare_means()`: argument is of length zero

**Comment by jsan4christ** [comment]
Hi, No worries, I have read from your other post that I need to restart R and doing so help. Hope it helps others who end up here and have the same problem.
